plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -664,7 +664,20 @@ def settings(module, proxmox, vmid, node, name, timeout, **kwargs):
         return False
 
 
-def create_vm(module, proxmox, vmid, newid, node, name, memory, cpu, cores, sockets, timeout, update, **kwargs):
+def wait_for_task(module, proxmox, node, taskid):
+    timeout = module.params['timeout']
+
+    while timeout:
+        task = proxmox.nodes(node).tasks(taskid).status.get()
+        if task['status'] == 'stopped' and task['exitstatus'] == 'OK':
+            time.sleep(1)
+            return True
+        time.sleep(1)
+        timeout = timeout - 1
```

Comment:
```suggestion
        timeout = timeout - 1
        if timeout == 0:
            break
        time.sleep(1)
```
Otherwise you wait an extra second.

Reply:
Ah, I see that you mentioned that in the PR description. I guess you should also add a comment and document that in the code (and also mention it in the changelog fragment).

Reply:
I added a comment in `wait_for_task()`. Please tell me if it is not enough.

Reply:
That comment is good. I don't think the additional wait on failure is necessary though, so I think the above suggestion still applies.